server/redis.repository.js

```diff
@@ -22,7 +22,7 @@ const initializeRoom = async (roomId, numGuests, movies) => {
     }
     let roomArgs = ["numUsers", numGuests, "total_swipes", 0, "numMovies", movies.length];
     movies.forEach(element => {
-        roomArgs.push(element.id);
+        roomArgs.push(element['netflixid']);
```

Comment:
Since the key is a string in the object, I don't think doing `element.netflixId` will work so I made it `element['netflixId]` to get the value of the netflixId key. I think this makes sense. What are your thoughts @dlhxu?

Reply:
```
{ 
download: "1"
image: "https://occ-0-2851-38.1.nflxso.net/dnm/api/v6/evlCitJPPCVCry0BZlEFb5-QjKc/AAAABXxvlp3diQwp7pLzNj-PiTLiMv7Kz-Sba-BElRYGLYbeCcrOpYuMnXnSublbCBzsInezU_9DGxHFqk0MmDcf9NtdRGIMCTwTPK5J-rIIQ_uV0OOiBikG4FZ638Q.jpg?r=b39"
imdbid: "tt2466830"
largeimage: "http://cdn0.nflximg.net/images/0146/11810146.jpg"
netflixid: "70290567"
rating: "7.4"
released: "2013"
runtime: "1h39m"
synopsis: "They are fiercely competitive athletes, determined to become champions on one of the world&#39;s toughest golf courses. And they&#39;re still in grade school."
title: "The Short Game"
type: "movie"
unogsdate: "2015-04-14"
}
```
I believe that element.netflixid will get you the id that you want, as this is what I see in the console if I console.log the returned top10 list of movies that we are fetching from the API. Give element.netflixid a try and let me know if it works out!

Reply:
Thanks!